ote/src/cljs/ote/style/base.cljs

```diff
@@ -51,9 +43,18 @@
                           ::stylefy/mode {:hover {:border-bottom (str "1px solid " colors/primary)
                                                   :margin-bottom "-1px"}}})
 
+(def base-button
```

Comment:
I would move these to buttons styles file

Reply:
Yes, but `base-button` is an old style which seems to be used only in one place anymore. These commits did not modify `base-button`.

It seems to be about the delete-button (bin icon) in service tables. I order to finish things, this task will update only action buttons and future tasks may improve layouts and styles more :)